keras/src/layers/convolutional/depthwise_conv1d.py

```diff
@@ -92,7 +92,7 @@ class DepthwiseConv1D(BaseDepthwiseConv):
     Example:
 
     >>> x = np.random.rand(4, 10, 12)
-    >>> y = keras.layers.DepthwiseConv1D(3, 3, 2, activation='relu')(x)
+    >>> y = keras.layers.DepthwiseConv1D(kernel_size=3, depth_multiplier=3, strides=2, activation='relu')(x)
```

Comment:
This doesn't pass the linter, every line should be below 80 characters. See below:
``
./keras/src/layers/convolutional/depthwise_conv1d.py:95:81: E501 line too long (108 > 80 characters)
``

Reply:
Hi @ruidazeng,
Changed .Thank you for the suggestion !